### PR TITLE
Make cjwagner the codeowner of prow/**

### PR DESCRIPTION
**Pull request by cjwagner**
/cc @BenTheElder

### CODE REVIEW

**Comment by BenTheElder** [comment]
@cjwagner:
I am the captain now

File: CODEOWNERS [hunk 1]
```diff
@@ -5,7 +5,7 @@ kettle/** @rmmh
 logexporter/** @shyamjvs
 mungegithub/** @cjwagner
 planter/** @bentheelder
-prow/** @spxtr
+prow/** @cjwagner
```
Comment (spxtr) [inline CODEOWNERS]: hey!

Comment (fejta) [reply CODEOWNERS]: Do you want to be added to all PRs still Joe?

Comment (spxtr) [reply CODEOWNERS]: No that's fine :)

**Pull request merged**